### DISCUSSION TITLE
purchase failed?

### DISCUSSION THREAD

**Discussion by sankarov** [community/community#145135] [Copilot Conversations]
I bought copilot on github mobile, but after paying, it is not active (purchase failed) and can't restore purchase, does anyone know why?![Screenshot_20241120-042626_GitHub.png](https://github.com/user-attachments/assets/20a3b461-3e8e-4fb4-9898-13c4c0be5283)

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @sankarov, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
